openehr-rm/src/main/java/com/nedap/archie/rm/support/identification/GenericId.java

```diff
@@ -1,10 +1,17 @@
 package com.nedap.archie.rm.support.identification;
 
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlType;
+
 /**
  * Created by pieter.bos on 08/07/16.
  */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "GENERIC_ID", propOrder = {"scheme"})
 public class GenericId extends ObjectId {
-
+    @XmlElement
```

Comment:
I would expect this to be the default. Any specific reason for adding @XmlElement here?